frappe/core/page/dashboard_view/dashboard_view.js

```diff
@@ -27,8 +27,11 @@ class Dashboard {
 		this.page = wrapper.page;
 	}
 
-	show() {
+	async show() {
 		this.route = frappe.get_route();
+		if (!locals.DocType["Dashboard"]) {
+			await frappe.model.with_doctype("Dashboard")
```

Comment:
This can be moved inside breakdcrumbs.js to always load the doctype without hardcoding it.